spring-boot-project/spring-boot/src/main/java/org/springframework/boot/orm/jpa/EntityManagerFactoryBuilder.java

```diff
@@ -104,18 +107,34 @@ public void setBootstrapExecutor(AsyncTaskExecutor bootstrapExecutor) {
 		this.bootstrapExecutor = bootstrapExecutor;
 	}
 
+	/**
+	 * Set the PersistenceUnitPostProcessors to be applied to the PersistenceUnitInfo used
+	 * for creating this EntityManagerFactory.
+	 * <p>
+	 * Such post-processors can, for example, register further entity classes and jar
+	 * files, in addition to the metadata read from {@code persistence.xml}.
+	 * <p>
+	 * <b>NOTE: Only applied if no external PersistenceUnitManager specified.</b>
+	 * {@link LocalContainerEntityManagerFactoryBean#setPersistenceUnitPostProcessors}
+	 * @param persistenceUnitPostProcessors internal persistence unit post processors
+	 * @since 2.5.0
+	 */
+	public void setPersistenceUnitPostProcessors(PersistenceUnitPostProcessor... persistenceUnitPostProcessors) {
+		this.persistenceUnitPostProcessors = persistenceUnitPostProcessors;
+	}
+
 	/**
 	 * A fluent builder for a LocalContainerEntityManagerFactoryBean.
 	 */
 	public final class Builder {
 
-		private DataSource dataSource;
+		private final DataSource dataSource;
```

Comment:
Thanks but we prefer that such unrelated change be done in an isolated commit. Can you please revert that?